pump/storage/storage.go

```diff
@@ -707,39 +739,21 @@ func (a *Append) doGCTS(ts int64) {
 	}()
 
 	for {
-		nStr, err := a.metadata.GetProperty("leveldb.num-files-at-level0")
-		if err != nil {
-			log.Error("get `leveldb.num-files-at-level0` property failed", zap.Error(err))
-			break
-		}
-
-		l0Num, err := strconv.Atoi(nStr)
+		l0Num, err := getL0Num()
 		if err != nil {
-			log.Error("parse `leveldb.num-files-at-level0` result to int failed", zap.String("str", nStr), zap.Error(err))
-			break
+			log.Error("", zap.Error(err))
```

Comment:
Empty message?

Reply:
It's @GregoryIan 's suggest.
He thinks that messages are already in L697 and L702